ui/app/templates/components/client-node-row.hbs

```diff
@@ -6,20 +6,17 @@
   {{/if}}
 </td>
 <td data-test-client-id>{{#link-to "clients.client" node.id class="is-primary"}}{{node.shortId}}{{/link-to}}</td>
-<td data-test-client-name class="is-200px is-truncatable" title="{{node.name}}">{{node.name}}</td>
-<td data-test-client-status>{{node.status}}</td>
-<td data-test-client-drain>
-  {{#if node.isDraining}}
-    <span class="status-text is-info">true</span>
-  {{else}}
-    false
-  {{/if}}
+<td class="is-200px is-truncatable is-hover-overflowable" title="{{node.name}}">
+  <span data-test-client-name>{{node.name}}</span>
+  <div class="hover-overflow" aria-hidden>{{node.name}}</div>
 </td>
-<td data-test-client-eligibility>
-  {{#if node.isEligible}}
-    {{node.schedulingEligibility}}
+<td data-test-client-status title={{statusTitle}}>
```

Comment:
There are some existing places where we use `aria-label` and the `.tooltip` class to [create immediate tooltips](https://github.com/hashicorp/nomad/blob/master/ui/app/templates/components/allocation-row.hbs#L31-L35) vs. the browser's delayed tooltip.

I don't think it will support multiline messages, but maybe the `statusTitle` could be reworked to fit in one line?

Maybe something like `{{status}} / {{if isDraining "draining" "not draining"}} / {{is isEligible "eligible" "not eligible"}}`?

Reply:
👍 nice, I definitely prefer tooltips to titles. I made this change but there’s a resulting z-index tragedy. I’ll see what I can do about that.